main.go

```diff
@@ -6,31 +6,45 @@ import (
 
 	"github.com/lidofinance/terra-monitors/app"
 	"github.com/lidofinance/terra-monitors/collector"
+	"github.com/lidofinance/terra-monitors/collector/monitors"
 	"github.com/lidofinance/terra-monitors/extractor"
 	"github.com/lidofinance/terra-monitors/internal/logging"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	HubContract            = "terra1mtwph2juhj0rvjz7dy92gvl6xvukaxu8rfv8ts"
+	RewardContract         = "terra17yap3mhph35pcwvhza38c2lkj7gzywzy05h7l0"
+	BlunaTokenInfoContract = "terra1kc87mu460fwkqte29rquh4hc20m54fxwtsx7gp"
```

Comment:
Isn't it better to store these variable in a config (with lcd endpoint)?. Cause these are mainnet values but the monitoring will also be for a testnet, no?

Reply:
like